engine/write_batch_impl.cpp

```diff
@@ -1,10 +1,17 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2022 Intel Corporation
+ */
+
 #include "write_batch_impl.hpp"
 
 #include "alias.hpp"
 
 namespace KVDK_NAMESPACE {
 
 void BatchWriteLog::EncodeTo(char* dst) {
```

Comment:
This function should not be called without PMem support. Please add TODO to mark this as a temporary workaround.